Makefile

```diff
@@ -214,22 +244,24 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(MAKE) docker-push OPERATOR_IMAGE=$(BUNDLE_IMG)
 
-.PHONY: opm
-OPM = ./bin/opm
-opm: ## Download opm locally if necessary.
-ifeq (,$(wildcard $(OPM)))
-ifeq (,$(shell which opm 2>/dev/null))
-	@{ \
-	set -e ;\
-	mkdir -p $(dir $(OPM)) ;\
-	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$${OS}-$${ARCH}-opm ;\
-	chmod +x $(OPM) ;\
-	}
-else
-OPM = $(shell which opm)
-endif
-endif
+.PHONY: fix-csv-replaces
+fix-csv-replaces: $(YQ)
+	$(eval REPLACES_VERSION=$(shell curl -sSL -H "Accept: application/vnd.github+json" \
+               -H "Authorization: Bearer ${GH_TOKEN}" \
+               -H "X-GitHub-Api-Version: 2022-11-28" \
+               https://api.github.com/repos/Kuadrant/authorino-operator/releases/latest | \
+               jq -r '.name'))
```

Comment:
This also works without the `GH_TOKEN`, doesn't it?

```sh
❯ curl -sSL \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/Kuadrant/authorino-operator/releases/latest | jq -r '.name'
v0.8.0
```

Reply:
Aha you're not wrong - we followed the docs described here which did not state the token as optional for public repos - https://docs.github.com/en/free-pro-team@latest/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release

If this is the case I'd prefer to opt for removing the `GH_TOKEN` altogether

Reply:
> I'd prefer to opt for removing the `GH_TOKEN`

I was thinking the same. The main (only?) benefit of the token here is to avoid getting rate-limited, but since this is for the release workflow, it's not like it's something that requires many requests anyway.

Reply:
Yep agreed, we'll update this

Reply:
Just pushed the changes as suggested. However, in the meantime I noticed the workflow failing when authorinoVersion parameter is left as "latest". Works fine when it is set to a specific version like 0.14.0 for example.